assembl/static2/js/app/utils/linkify.js

```diff
@@ -8,8 +8,14 @@ function addIframeForStoryChief(html: string): string {
   return html.replace(url, '<iframe class="synthesis-iframe" src="$2"></iframe>');
 }
 
-export function transformLinksInHtml(html: string): string {
+const HASHTAG_REGEX = /(\s|^|\n|>)(#[A-Za-z0-9\-_]+)/g;
```

Comment:
I'm not familiar with the caret in between two pipes, what's the significance of this?

Reply:
i mean here : a space, or *the start of the string* or a new line or a ">"